vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -174,121 +225,76 @@ void WindowsModalHostComponentView::RegisterWndClass() noexcept {
   registered = true;
 }
 
+winrt::Microsoft::ReactNative::Composition::Experimental::IVisual
+WindowsModalHostComponentView::VisualToMountChildrenInto() noexcept {
+  return m_reactNativeIsland
+      .as<winrt::Microsoft::ReactNative::Composition::Experimental::IInternalCompositionRootView>()
+      .InternalRootVisual();
+}
+
+// childComponentView - reference to the child component view
+// index - the position in which the childComponentView should be mounted
 void WindowsModalHostComponentView::MountChildComponentView(
     const winrt::Microsoft::ReactNative::ComponentView &childComponentView,
     uint32_t index) noexcept {
-  // Disabled due to partial Modal implementation. Tracking re-enablement with task list here:
-  // https://github.com/microsoft/react-native-windows/issues/11157 assert(false);
+  EnsureModalCreated();
   base_type::MountChildComponentView(childComponentView, index);
 }
 
 void WindowsModalHostComponentView::UnmountChildComponentView(
     const winrt::Microsoft::ReactNative::ComponentView &childComponentView,
     uint32_t index) noexcept {
-  // Disabled due to partial Modal implementation.Tracking re-enablement with task list here : https : //
-  // github.com/microsoft/react-native-windows/issues/11157 assert(false);
   base_type::UnmountChildComponentView(childComponentView, index);
 }
 
-void WindowsModalHostComponentView::HandleCommand(
-    const winrt::Microsoft::ReactNative::HandleCommandArgs &args) noexcept {
-  Super::HandleCommand(args);
+void WindowsModalHostComponentView::updateLayoutMetrics(
+    facebook::react::LayoutMetrics const &layoutMetrics,
+    facebook::react::LayoutMetrics const &oldLayoutMetrics) noexcept {
+  base_type::updateLayoutMetrics(layoutMetrics, oldLayoutMetrics);
+  if (m_hwnd) {
+    EnsureModalCreated();
+    AdjustWindowSize();
```

Comment:
So anytime the contents change desired size, this will snap the window back to that size.  Given that, we should probably disable the ability for the user to resize the hwnd?

Try adding a button that changes the size of the content within the modal, and see what affects you get with manual resize + having the UI within the modal change size.

Reply:
Currently manually resizing the hwnd doesn't do anything since I haven't added the WM_WINDOWPOSCHANGED message yet, it doesn't hit anything in this file and just opens to a blank white screen. Having the content changed does snap it back to the new size though!

Will disable the ability for the user to resize the window though! 

See here: 
![devenv_CAFkJWBPvB](https://github.com/user-attachments/assets/917dae89-f4d5-4cc4-955c-814ecfadeb43)

Reply:
Discussed online, disabled the user-input for now!